docs/resources/cassandra.md

```diff
@@ -77,18 +77,18 @@ resource "aiven_cassandra" "bar" {
 
 Optional:
 
-- `additional_backup_regions` (List of String) Additional Cloud Regions for Backup Replication
-- `cassandra` (Block List, Max: 1) cassandra configuration values (see [below for nested schema](#nestedblock--cassandra_user_config--cassandra))
-- `cassandra_version` (String) Cassandra major version
-- `ip_filter` (List of String, Deprecated) Allow incoming connections from CIDR address block, e.g. '10.20.0.0/16'
-- `ip_filter_object` (Block List, Max: 1024) Allow incoming connections from CIDR address block, e.g. '10.20.0.0/16' (see [below for nested schema](#nestedblock--cassandra_user_config--ip_filter_object))
+- `additional_backup_regions` (List of String) Additional Cloud Regions for Backup Replication.
+- `cassandra` (Block List, Max: 1) Cassandra configuration values. (see [below for nested schema](#nestedblock--cassandra_user_config--cassandra))
```

Comment:
Doesn't play well with "(see ..)

Reply:
since the `(see ...)` part is added by the Terraform docs generator itself, this would be unreasonably hard to catch and drop the dot, and I don't think that this worths the effort